packages/instant-apps-components/src/components/instant-apps-social-share/instant-apps-social-share.test.ts

```diff
@@ -0,0 +1,300 @@
+import { expect, test, describe } from 'vitest';
+
+import '../../../dist/components/instant-apps-social-share.js';
+
```

Comment:
extra spacing here. might be a good idea to set up a code formatter. Try looking into `prettier`